### PR TITLE
Add basic tutorial tasks

### PR DESCRIPTION
**Pull request by MilBia**
I add some tutorial tasks with only placeholders texts based on ours examples and tests.

To test everything before  commit be merged with master change `start_slurm` function to following:
```
function start_slurm() {
    cd "$JUPYTER_CONTAINERS_DIR/docker_config/slurm"
      ./start-slurm.sh
    cd -

    docker exec slurmctld /bin/bash -c "conda install -c conda-forge jupyterlab"
    docker exec slurmctld /bin/bash -c "conda install -c conda-forge distributed"
    docker exec slurmctld /bin/bash -c "conda install -c conda-forge nodejs"
    docker exec slurmctld /bin/bash -c "pip install dask_labextension"
    docker exec slurmctld /bin/bash -c "cd /jobqueue_features; pip install -r requirements.txt; pip install --no-deps -e ."
    docker exec c1 /bin/bash -c "cd /jobqueue_features; pip install -r requirements.txt; pip install --no-deps -e ."
    docker exec c2 /bin/bash -c "cd /jobqueue_features; pip install -r requirements.txt; pip install --no-deps -e ."

    cd "$JUPYTER_CONTAINERS_DIR/docker_config/slurm"
      docker cp tutorial_tasks/ c1:/jobqueue_features/tutorial/docker_config/slurm/tutorial_tasks
      docker cp tutorial_tasks/ c2:/jobqueue_features/tutorial/docker_config/slurm/tutorial_tasks
      docker cp tutorial_tasks/ slurmctld:/jobqueue_features/tutorial/docker_config/slurm/tutorial_tasks
    cd -

    docker exec slurmctld /bin/bash -c "adduser slurmuser; chown -R slurmuser /jobqueue_features;"
    docker exec c1 /bin/bash -c "adduser slurmuser; chown -R slurmuser /jobqueue_features;"
    docker exec c2 /bin/bash -c "adduser slurmuser; chown -R slurmuser /jobqueue_features;"
    docker exec slurmctld /bin/bash -c "yes|sacctmgr create account slurmuser; yes | sacctmgr create user name=slurmuser Account=slurmuser"
    docker exec slurmctld /bin/bash -c "mkdir -p /home/slurmuser/.config/dask/"
    cd "$JUPYTER_CONTAINERS_DIR/docker_config/slurm"
      docker cp labextension.yaml slurmctld:/home/slurmuser/.config/dask/labextension.yaml
    cd -
    docker exec -u slurmuser slurmctld /bin/bash -c "chmod u=rw,go=r /home/slurmuser/.config/dask/labextension.yaml"
    docker exec -u slurmuser slurmctld /bin/bash -c "cd /jobqueue_features/tutorial/docker_config/slurm/tutorial_tasks; jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root --NotebookApp.token='' --NotebookApp.password='' --NotebookApp.notebook_dir='/jobqueue_features/tutorial/docker_config/slurm/tutorial_tasks'&"

    echo
    echo -e "\e[32mSLURM properly configured\e[0m"
    echo
    echo -e "\tOpen your browser at http://localhost:8888/lab"
    echo
}
```
After merge everything should work withouht it.

### CODE REVIEW

**Comment by ocaisa** [comment]
There is an annoying test failure here not to do with this PR.

**Comment by MilBia** [comment]
I assume the failure is related to update of docker images we are using. Changes delete form base image `mpicc` which is included in our tests. I continue investigating that.

**Comment by ocaisa** [comment]
mpicc should come from the mpich package, not sure why these are being uninstalled

**Comment by ocaisa** [comment]
The mpich package was updated a few days ago, not sure what changed

File: ci/pbs/Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ ENV PBS_MASTER pbs-master
 RUN yum install -y which && yum clean all
 
 # Install MPICH and mpi4py (and black)
-RUN conda install --yes -c conda-forge black mpich mpi4py && conda clean -tipy
+RUN conda install --yes -c conda-forge black mpi4py && conda install --yes -c conda-forge/label/cf202003 mpich  && conda clean -tipy
```
Comment (ocaisa) [inline ci/pbs/Dockerfile]: The other option here (that I prefer) is to pin the mpich version:
```suggestion
RUN conda install --yes -c conda-forge black mpich=3.3.2 mpi4py && conda clean -tipy
```

Comment (ocaisa) [inline ci/pbs/Dockerfile:10]: The other option here (that I prefer) is to pin the mpich version:
```
RUN conda install --yes -c conda-forge black mpich=3.3.2 mpi4py && conda clean -tipy
```

**Comment by ocaisa** [comment]
CI is passing so let's merge this so we can test it

**Pull request merged**